bigchaindb/models.py

```diff
@@ -118,8 +118,36 @@ def validate(self, bigchain):
 
 
 class Block(object):
+    """A Block bundles up to 1000 Transactions. Nodes vote on its validity.
```

Comment:
If we make the number of transactions per block customizable we need to remember changing this number as well. I'd remove the precise number (`1000`) from the docstring.